File: Bai_79/main.py
def calculate_scissor_length(n, m):
    # Tính tổng số cạnh của hình chữ nhật
    edges = 2 * (n + m)
    # Độ dài đường cắt là tổng số cạnh trừ đi số ô trong hình chữ nhật
    scissor_length = edges - n * m
    return scissor_length

# Đọc dữ liệu từ file input
with open('SCISSOR.INP', 'r') as f:
    n, m = map(int, f.readline().split())

# Tính độ dài đường cắt
result = calculate_scissor_length(n, m)

# Ghi kết quả ra file output
with open('SCISSOR.OUT', 'w') as f:
    f.write(str(result))

File: Bai_79/main.cpp
#include <iostream>
using namespace std;

int main() {
    // Mở file input và output
    freopen("SCISSOR.INP", "r", stdin);
    freopen("SCISSOR.OUT", "w", stdout);

    // Đọc dữ liệu vào từ file input
    int n, m;
    cin >> n >> m;

    // Tính tổng độ dài đường cắt
    int total_length = 0;
    for (int i = 0; i < m; ++i) {
        total_length += n; // Đường cắt dọc
        if (i % 2 == 0) {
            total_length += (n - 1); // Đường cắt ngang
        }
    }

    // In kết quả ra file output
    cout << total_length << endl;

    return 0;
}
